submissions/kt-std/html-css-popup/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="css/style.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;&display=swap" rel="stylesheet"> 
+	<title>HTML & CSS popup</title>
+</head>
+<body class="column">
+	<header>
+		<nav class="nav">
+			<ul class="nav__list">
+				<li class="nav__item box flex-center"><a href="#mail" class="item__link flex-center column box">Mail</a></li>
+				<li class="nav__item box flex-center"><a href="#pictures" class="item__link flex-center column box">Pictures</a></li>
+				<li class="nav__item box flex-center popup-btn">
+					<input type="checkbox" class="checkbox__input" id="check">
+					<label class="checkbox flex-center full" for="check">
+						<img src="assets/list.svg" alt="List of other applications" class="checkbox__image">
+					</label>					
+					<ul class="popup">
+						<li class="popup__item item">
```

Comment:
You have unused class `item` here.

Reply:
Thank you reviewing! 
I've deleted unused classes, changed the color of hovering items so that they match each other, and formatted the code.